test/spec/additional-data-spec/adjust_date-01.spec.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint max-len: 0 */
```

Comment:
Instead of `additional-data-spec`, perhaps we can use `sparql-12/sep-0002`.

Reply:
I agree!